### DISCUSSION TITLE
How can I disable the multi-root option?

### DISCUSSION THREAD

**Discussion by mrsdz** [emacs-lsp/lsp-mode#3298] [Q&A] [answered]
Hi, thanks for this awesome project! I'm a complete emacs-lsp newbie and I want to know how can I disable the multi-root feature? I want to lsp-mode (pyright) just watch the current project. but every time after switching to a project, a message shows that it's watching 2 or 3 projects.

<img width="406" alt="image" src="https://user-images.githubusercontent.com/18589581/148064843-92dd01c5-76b9-4882-b78a-df30e268f68d.png">

**Comment by yyoncho** [comment, marked as answer]
There is `lsp-pyright-multi-root`  (has to be set before lsp-mode is loaded).

**Reply by mrsdz** [reply]
Would you please help me with how can I set this variable before lsp-mode?

**Reply by yyoncho** [reply]
Put `(setq lsp-pyright-multi-root t)` on top of your init file.

**Reply by mrsdz** [reply]
It didn't work :(
I'm using doom emacs and  I put this variable on the top of my `~/.doom/init.el` but didn't work.

**Reply by yyoncho** [reply]
Sorry, it should be `(setq lsp-pyright-multi-root nil)`

**Reply by mrsdz** [reply]
Thank you, it works.
